lib/function/matrix/sqrtm.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+function factory(type, config, load, typed) {
+  var matrix   = load(require('../../type/matrix/function/matrix'));
+  var abs      = load(require('../arithmetic/abs'));
+  var add      = load(require('../arithmetic/add'));
+  var divide   = load(require('../arithmetic/divide'));
+  var multiply = load(require('../arithmetic/multiply'));
+  var sqrt     = load(require('../arithmetic/sqrt'));
+  var subtract = load(require('../arithmetic/subtract'));
+  var inv      = load(require('../matrix/inv'));
+  var size     = load(require('../matrix/size'));
+  var max      = load(require('../statistics/max'));
+  var eye      = load(require('./eye'));
+
+  /**
+   * Calculate the principal square root of a square matrix.
+   * The principal square root matrix `X` of another matrix `A` is such that `X * X = A`.
+   *
+   * https://en.wikipedia.org/wiki/Square_root_of_a_matrix
+   * 
+   * Syntax:
+   *
+   *     X = math.sqrtm(A)
+   *
+   * Examples:
+   *
+   *     math.sqrtm([[1, 2], [3, 4]]); // returns [[-2, 1], [1.5, -0.5]]
+   *
+   * See also:
+   *
+   *     det, transpose
```

Comment:
Are these the best suggestions? I would say replace these with more relavant functions.

Reply:
This was a copy-paste issue. However, I am not so sure as to which ones to list here... What do you think of listing `sqrt`, `max`, and maybe `abs`?

Reply:
Put `pow` in aswell as `math.pow(math.sqrtm(A))` should equal `A` (might be worth testing for this)